### ISSUE TITLE
`n:tag` doesn't work when combined with `n:ifcontent`

### ISSUE THREAD

**Issue by zipper** [nette/latte#288]
Version: 2.11.2

### Bug Description
If you use both n:tag and n:ifcontent on same element, n:tag is ignored and doesn't work.

### Steps To Reproduce
First `<div>` is correctly changed to `<p>`, the one second isn't.
```latte
{var $foo = 'p'}
<div n:tag="$foo">foo</div>
<div n:ifcontent n:tag="$foo">foo</div>
```

### Expected Behavior
Tag should be changed properly to `$tagName` even when `n:ifcontent` is used. In real life example, I'm also using `n:inner-block` on the element, therefore it might be empty (and not outputted at all).

**Comment by zipper** [comment]
Also doesn't work with `n:if` which I have just tried to use as a workaround.

**Comment by dg** [comment]
It's fixed in Latte 3 5543489f2c582d3d2e8d1909632dac9f3ff6affa

I won't fix it in the older version.
